### PR TITLE
Change 'Vote' to 'Proposal'

### PR DESCRIPTION
**Pull request by babasholaking**
Change 'Vote' to 'Proposal' and all refences to proposal 'id' to 'proposalId'

### CODE REVIEW

File: contracts/AventusVote.sol [hunk 1]
```diff
@@ -58,79 +58,79 @@ contract AventusVote is Owned {
...
-    onlyVoteCreator(id)
+    onlyProposalCreator(proposalId)
   {
-    LVote.addVoteOption(s, id, option);
+    LVote.addProposalOption(s, proposalId, option);
   }
 
-  /** 
+  /**
   * @dev Finish setting up votes with time intervals & start
```
Comment (andy-mcewan-aventus-io) [inline contracts/AventusVote.sol]: Finish setting up votes => Finish setting up proposal

File: contracts/AventusVote.sol [hunk 2]
```diff
@@ -58,79 +58,79 @@ contract AventusVote is Owned {
...
-  * @param id Proposal ID
+  * @param proposalId Proposal ID
   * @param start The start date of the cooldown period, after which vote starts
```
Comment (andy-mcewan-aventus-io) [inline contracts/AventusVote.sol]: vote => voting on proposal

or just "voting"

Comment (babasholaking) [reply contracts/AventusVote.sol]: Fixed!

[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/AventusVote.sol]: fixed!!

File: contracts/libraries/LVote.sol [hunk 3]
```diff
@@ -1,71 +1,71 @@
...
+    s.setUInt(keccak256("Proposal", proposalId, "OptionsCount"), count + 1);
   }
 
-  /** 
+  /**
   * @dev Finish setting up votes with time intervals & start
   * @param s Storage contract
-  * @param id Proposal ID
+  * @param proposalId Proposal ID
   * @param start The start date of the cooldown period, after which vote starts
```
[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/libraries/LVote.sol]: Updated!!

File: contracts/libraries/LVote.sol [hunk 4]
```diff
@@ -1,71 +1,71 @@
...
     // Store new options count, and the option + description
-    s.setString(keccak256("Vote", id, "option", count + 1), option);
-    s.setUInt(keccak256("Vote", id, "OptionsCount"), count + 1);
+    s.setString(keccak256("Proposal", proposalId, "option", count + 1), option);
+    s.setUInt(keccak256("Proposal", proposalId, "OptionsCount"), count + 1);
   }
 
-  /** 
+  /**
   * @dev Finish setting up votes with time intervals & start
```
[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/libraries/LVote.sol]: Updated

File: contracts/libraries/LVote.sol [hunk 5]
```diff
@@ -1,71 +1,71 @@
...
   {
     uint voteCount = s.getUInt(keccak256("VoteCount"));
-    uint id = voteCount + 1;
+    uint proposalId = voteCount + 1;
 
-    s.setString(keccak256("Vote", id, "description"), desc);
-    s.setUInt(keccak256("VoteCount"), id);
-    s.setAddress(keccak256("Vote", id, "creator"), creator);
+    s.setString(keccak256("Proposal", proposalId, "description"), desc);
+    s.setUInt(keccak256("VoteCount"), proposalId);
```
[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/libraries/LVote.sol]: Fixed!!

File: contracts/libraries/LVote.sol [hunk 6]
```diff
@@ -1,71 +1,71 @@
...
+  * @return uint proposalId of newly created proposal
   */
-  function createVote(IStorage s, address creator, string desc)
+  function createProposal(IStorage s, address creator, string desc)
     public
     returns (uint)
   {
     uint voteCount = s.getUInt(keccak256("VoteCount"));
-    uint id = voteCount + 1;
+    uint proposalId = voteCount + 1;
```
[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/libraries/LVote.sol]: Updated

File: contracts/AventusVote.sol [hunk 7]
```diff
@@ -58,79 +58,79 @@ contract AventusVote is Owned {
...
-  function revealVote(uint id, uint optId, uint8 v, bytes32 r, bytes32 s_) public {
-    LVote.revealVote(s, id, optId, v, r, s_);
+  function revealVote(uint proposalId, uint optId, uint8 v, bytes32 r, bytes32 s_) public {
+    LVote.revealVote(s, proposalId, optId, v, r, s_);
   }
 
 
-  /** 
+  /**
   * @dev Upgrade Storage if necissary
```
[the remark this answers is not in the document]
Comment (babasholaking) [reply contracts/AventusVote.sol]: Updated!!

**Pull request merged**